augur/export_v2.py

```diff
@@ -603,13 +613,16 @@ def run_v2(args):
 
     add_metadata_to_tree(auspice_json["tree"], node_metadata)
 
+    mutations_present = False
+    if check_muts(node_metadata):
+        mutations_present = True
```

Comment:
why not
`mutations_present = bool(check_muts(node_metadata))`

Reply:
and move it to line 625 😉